src/index.ts

```diff
@@ -368,6 +368,14 @@ const getBundleName = (bundle: Record<string, any>): string => {
     return '';
 }
 
+const checkForBom = (module: string):boolean => {
+    if (module.startsWith('\0')) {
```

Comment:
Is there a scenario where this will return a false positive and block a character we actually do want to accept?

Reply:
I don't think so, '\0' is supposed to be a regex null char so I don't think a string that we'd want to accept could match this pattern. But to make this safer I could turn this straight up into a string search by regex, that could eliminate any false positives and could expand coverage to past the first char in the string as well.